x86/proofs/decode.ml

```diff
@@ -1624,6 +1641,7 @@ let READ_SIB_CONV,READ_MODRM_CONV,READ_VEX_CONV,DECODE_CONV =
     evaluate (rhs (concl th)) (F o TRANS th)
   | Comb(Const("@",_),_) -> failwith "ARB"
   | Const("ARB",_) -> failwith "ARB"
+  | Comb(Comb((Const("word_shl",_) as f),a),b) -> eval_binary f a b F WORD_SHL_CONV
```

Comment:
The evaluator needs to understand `word_shl` and `NUMERAL` for the REX.R code in `read_VEX`.